File: Python/01 - Introduction to Programming/Task17/alternative.py
#Task17 - alternative.py
#------------------------------------------------------------#
#request input from user
original = input("Please enter a value for a string variable: ")
alt = ""

# make each alternate character an uppercase character and each other alternate character a lowercase character.
for i in range (0, len(original)):
	if(i%2==0):
		alt += original[i].upper()
	else:
		alt += original[i].lower()

#display result
print(alt)

File: Ruby/01 - Introduction to Programming/Task17/alternative.rb
#Task17 - alternative.rb
#------------------------------------------------------------#
#request input from user
print "Please enter a value for a string variable: "
original = gets.chomp()
#alt = ""

# make each alternate character an uppercase character and each other alternate character a lowercase character.
for i in Range.new(0, original.length)
	if i%2==0
		alt = alt + original[i].upcase #********************************************
	else
		alt = alt + original[i].downcase #******************************************
    end
end

#display result
puts alt